src/tasks/ShopSpecialTask.py

```
@@ -58,29 +58,29 @@ def do_run(self):
         self.info_set("成功次数", "0")
         self.info_set("失败次数", 0)
         self.info_set("失败原因", None)
-        self.log_info(f"开始店长特供，共 {rounds} 轮")
+        self.log_info(self.tr("开始店长特供，共 {} 轮").format(rounds))
```

Ruff (0.15.13)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)